Firestore/core/src/firebase/firestore/remote/serializer.cc

```diff
@@ -16,8 +16,8 @@
 
 #include "Firestore/core/src/firebase/firestore/remote/serializer.h"
 
-#include <pb_decode.h>
-#include <pb_encode.h>
+#include "external/nanopb/src/nanopb/pb_decode.h"
```

Comment:
On xcode, #include <nanopb/pb*.h> works (as does #include <pb*.h>)

I'm not aware of any package builds either (though I didn't look beyond my work and personal desktops which are both apt based) but a make install would put pb*.h into /usr/include (or possibly /usr/local/include or more generally $prefix/include). All of nanopbs docs use #include <pb*.h>. (And rather notably, the nanopb *generated* code uses #include <pb*.h>.) Of course, there's nothing stopping debian/etc to put the headers into /usr/include/nanopb_or_anywhere_else_really anyways, and setup the include paths via pkg-config (or to not do that and include adjusted samples as part of the .deb.)

I think FindNanopb.cmake could be adjusted to alter the paths such that #include <nanopb/pb*.h> works. But then the generated code would break (unless we added some sort of fixup pass.)

tl;dr: I'm switching back to #include <pb*.h>. Alternative: use <nanopb/pb*.h> and run a fixup pass on the generated code.